Library/Homebrew/utils/repology.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "utils/curl"
+require "formula_info"
+
+module RepologyParser
+  module_function
+
+  def query_api(last_package_in_response = "")
+    url = "https://repology.org/api/v1/projects/#{last_package_in_response}?inrepo=homebrew&outdated=1"
+    ohai "Calling API #{url}" if Homebrew.args.verbose?
+
+    output, errors, status = curl_output(url.to_s)
+    output = JSON.parse(output)
+  end
+
+  def parse_api_response()
+    ohai "Querying outdated packages from Repology"
+    page_no = 1
+    ohai "Paginating repology api page: #{page_no}" if Homebrew.args.verbose?
+
+    outdated_packages = query_api()
+    last_pacakge_index = outdated_packages.size - 1
+    response_size = outdated_packages.size
+    page_limit = 15
+
+    while response_size > 1 && page_no <= page_limit
+      page_no += 1
+      ohai "Paginating repology api page: #{page_no}" if Homebrew.args.verbose?
+
+      last_package_in_response = outdated_packages.keys[last_pacakge_index]
+      response = query_api("#{last_package_in_response}/")
```

Comment:
Feels weird to have to pass the `/` here. That should be done by the called function, I think.